src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,54 @@
+import { useContext } from 'react';
+import { Status } from '../types/Status';
+
+import { TodoContext } from './TodosContext';
+import { TodoContextProps } from '../types/TodoContextProps';
+
+interface PropsFilter {
+  filter: Status;
+  setFilter: (filter: Status) => void;
+}
+
+export const TodosFilter: React.FC<PropsFilter> = () => {
+  const todoContext = useContext<TodoContextProps>(TodoContext);
+
+  const filter = todoContext?.filter || Status.ALL;
+  const setFilter = todoContext?.setFilter || (() => {});
```

Comment:
you don't need additional check `||`, as far as I see you have `Status.ALL` and `() => {}` by default in `filter`  and `setFilter`

just use destructuring as you did before
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/91946036/f543a392-1189-4cee-827d-7dfd79ae25f7)